handlers/handlers_test.go

```diff
@@ -270,7 +271,7 @@ func initialiseMockCookiesPageModel(cfg *config.Config, policy cookies.Policy, i
 	page.SiteDomain = cfg.SiteDomain
 	page.PatternLibraryAssetsPath = cfg.PatternLibraryAssetsPath
 	page.PreferencesUpdated = hasSetPreference
-	page.FeatureFlags.SixteensVersion = "67f6982"
+	page.FeatureFlags.SixteensVersion = mapper.SixteensVersionStr
```

Comment:
No value in setting this to a constant from a different package. I would argue that func `initialiseMockCookiesPageModel` is a mapper test and not needed in `handlers`

Reply:
My change undone.